lib/git_reflow/git_server/bit_bucket/pull_request.rb

```diff
@@ -78,6 +78,24 @@ def approvals
           approved
         end
 
+        def merge_feature_branch(feature_branch_name, options = {})
```

Comment:
I think this should be the default implementation and live in the base pull request (`lib/git_reflow/git_server/pull_request.rb`).  Also, I think it makes more sense to name this `merge!` (see below for why I'd like to see the bang)